openpype/hosts/blender/api/ops.py

```diff
@@ -1082,10 +1083,13 @@ def execute(self, context):
                     container.outliner_entity.children, substitute_collection
                 )
 
-        # Unlink entity from scene
+        # Move objects to either substituted collection or the parent one
         link_to_collection(
-            container.outliner_entity.objects, parent_collection
+            container.outliner_entity.objects,
```

Comment:
If `not isinstance(container.outliner_entity, bpy.types.Collection)` container.outliner_entity.objects could raise an AttributeError ?

Reply:
You're right, I've inverted `objects` and `children`.